Bikeini/components/Item.js

```diff
@@ -71,8 +75,24 @@ export default class Item extends React.PureComponent {
             {model}
           </Text>
         </View>
-        <Image style={styles.commentsTag} source={commentIcon} />
-        <Image style={styles.locationTag} source={locationIcon} />
+        <View style={styles.buttonRow}>
+          <TouchableOpacity
+            style={styles.commentsTag}
+            onPress={() => {
+              bikeData.showComments = true;
+              navigation.navigate('BikeInformation', { data: bikeData });
+            }}
+          >
+            <Image style={styles.commentsTag} source={commentIcon} />
+
+          </TouchableOpacity>
+          <TouchableOpacity
+            style={styles.locationTag}
+            onPress={() => { Alert.alert('Position me biatch'); }}
```

Comment:
Remove biatch, not the greatest word here...

Reply:
pc-fever